packages/ui/src/components/Chip.svelte

```diff
@@ -0,0 +1,99 @@
+<!--
+// Copyright © 2024 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+<script lang="ts">
+  import { createEventDispatcher } from 'svelte'
+  import { IconClose } from '..'
+
+  const dispatch = createEventDispatcher()
+
+  let buttonRef: HTMLButtonElement
+
+  function handleBackspace (event: KeyboardEvent) {
+    if (event.key === 'Backspace') {
+      dispatch('click')
+    }
+  }
+
+  export function focus () {
+    buttonRef.focus()
+  }
+</script>
+
+<div class="flex items-center font-medium-14 max-w-60 chip">
+  <span>
+    <slot/>
+  </span>
+  <button
```

Comment:
Can we use our `ModernButton`/`ButtonIcon`  component?

Reply:
They have different hover and focus colors for some reason…